src/test/java/HomeWorkApp.java

```diff
@@ -0,0 +1,48 @@
+public class HomeWorkApp {
+
+    public static void main(String[] args) {
+
+        printThreeWords();
+        checkSumSign();
+        printColor();
+        compareNumbers();
+    }
+
+    public static void printThreeWords() {
+        System.out.println("Orange"); // Апельсин
+        System.out.println("Banana"); // Банан
+        System.out.println("Apple");  // Яблоко
+    }
+
+    public static void checkSumSign() {
+        int a = 50;
+        int b = 20;
+        if (a + b >= 0) {
+            System.out.println("Сумма положительная.");
+        } else {
+            System.out.println("Сумма отрицательная.");
+        }
+    }
+
+    public static void printColor() {
+        int value = 50;
+        if (value <= 0) {
+            System.out.println("Red");
+        } else if (value > 0 && value <= 100) {
```

Comment:
```suggestion
        } else if (value <= 100) {
```
поскольку условие выше уже исключает данную проверку на 0